src/nvim/ex_docmd.c

```diff
@@ -1704,11 +1704,6 @@ static char_u * do_one_cmd(char_u **cmdlinep,
              /* Do not allow register = for user commands */
```

Comment:
Could this be dedented, or do you think it's out of scope? I say that because things like this seem really odd to me:
```
 if (       (ea.argt & REGSTR)
```